visualizations/frontend/filters/v2/TDatePicker.vue

```diff
@@ -58,6 +58,18 @@ export default {
             type: String,
             default: " ~ ",
         },
+        defaultStartDate: {
+            type: String,
+            default: "",
+        },
+        defaultEndDate: {
+            type: String,
+            default: "",
+        },
+        defaultDatePreset: {
+            type: String,
+            default: "",
+        }
```

Comment:
What potential values could user supply for `defaultDatePreset`?

Reply:
It should be the value that matches with your provided `dateFormat` prop, if format is `YYYY-MM-DD` then default should be `2022-12-31`

Reply:
so `defaultDatePreset` would be `2022-12-13`?

Reply:
Oh wait sorry totally missed it, no actually this one is a bit hard without documentation but this should just match the preset keys or labels.

- defaultDatePreset="ytd"

Reply:
Both will work, so any user can open dropdown, see what presets are available and type it's name, they do not have to know they key at all.

Reply:
Okey dokey. Yeah, we might need to document it... Or perhaps the `defaultDatePresets` could be an enum sometime in the future 🤷‍♀️  Let's see if it evolves